content/javascript/concepts/objects/terms/isExtensible/isExtensible.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: '.isExtensible()'
+Description: 'Returns a boolean value depending on whether a given object can have properties added to it.'
+Subjects:
+  - 'Web Development'
```

Comment:
```suggestion
  - 'Web Development'
  - 'Code Foundation'
```

Reply:
Thanks! I made a slight adjustment to your suggestion, 'Code Foundations' with an "s" [from the list here](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md).